src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -58,7 +57,7 @@ public static double getShooterHoodSetpoint() {
   }
 
   public static double getShooterHoodPositionTolerance() {
-    return getDouble("SHOOTER_HOOD_POSITION_TOLERANCE", SHOOTER_HOOD_POSITION_TOLERANCE);
+    return 2;
```

Comment:
Umm, what's the point of using the preferences if you're just going to adjust it in code?